### ISSUE TITLE
IBeX-GT: Insert space after autocompletion of EReferences

### ISSUE THREAD

**Issue by RolandKluge** [eMoflon/emoflon-ibex-democles#69]
The autocompletion for 'link variables' inserts the EReference name. This leads to the effect that pressing ctrl+space afterwards leads to the first suggestion being the EReference again. This could be resolved by inserting a space upon automcompletion.

How to reproduce (using handbook example):
* Create pattern with variable next : Field {}
* Inside the braces press, add "-"
* Press ctrl+space and select "figure"
* Press ctrl+space again. Now, I would expect to see only "->", but "figure" appears, first.

**Comment by patrickrobrecht** [comment]
I agree that one expects to see only `->`, but I didn't find a way to automatically add a space when a autocompletion proposal for a reference is accepted.

**Comment by RolandKluge** [comment]
Could the following hint help?

https://stackoverflow.com/questions/31423535/how-to-write-custom-autocompletion-in-xtext/31438040#31438040

**Comment by anthonyanjorin** [comment]
Any progress?  Have you been able to reproduce?  Is it clear what this issue means?
